src/xcode/ENA/ENA/Source/Scenes/Home/HomeInteractor.swift

```diff
@@ -295,21 +312,42 @@ final class HomeInteractor {
 	}
 
 	private func indexPathForActiveCell() -> IndexPath? {
-		let index = cells.firstIndex { cellConfigurator in
-			cellConfigurator === self.activeConfigurator
+		for section in sections {
+			let index = section.cellConfigurators.firstIndex { cellConfigurator in
+				cellConfigurator === self.activeConfigurator
+			}
+			guard let item = index else { return nil }
+			let indexPath = IndexPath(item: item, section: HomeViewController.Section.actions.rawValue)
+			return indexPath
 		}
-		guard let item = index else { return nil }
-		let indexPath = IndexPath(item: item, section: HomeViewController.Section.actions.rawValue)
-		return indexPath
+		return nil
 	}
 
 	private func indexPathForRiskCell() -> IndexPath? {
-		let index = cells.firstIndex { cellConfigurator in
-			cellConfigurator === self.riskConfigurator
+		for section in sections {
+			let index = section.cellConfigurators.firstIndex { cellConfigurator in
+				cellConfigurator === self.riskConfigurator
+			}
+			guard let item = index else { return nil }
+			let indexPath = IndexPath(item: item, section: HomeViewController.Section.actions.rawValue)
+			return indexPath
 		}
-		guard let item = index else { return nil }
-		let indexPath = IndexPath(item: item, section: HomeViewController.Section.actions.rawValue)
-		return indexPath
+		return nil
+	}
+
+	func showPinkCell() {
```

Comment:
please just remove the pink cell demo again before merging. Thats all, thanks again for improving the code base! :)

Reply:
I removed sample code

Reply:
But I like ping. :(